libs/search-widget/src/viewer/Entities.svelte

```diff
@@ -26,47 +53,29 @@
     if (event.key === 'Enter') search(entity);
   };
 
-  const getColor = (group: string) => {
-    const colors: { [key: string]: string } = {
-      DATE: '#ff8989',
-      FAC: '#81d8ac',
-      GPE: '#454ade',
-      LAW: '#1E264F',
-      LOC: '#b7a38d',
-      MAIL: '#e81c66',
-      ORG: '#6eb0ec',
-      PERSON: '#ffe186',
-      QUANTITY: '#b035c9',
-      MONEY: '#ff8c4b',
-      PERCENT: '#1e264f',
-      TIME: '#21b8a6',
-      EVENT: '#cba2da',
-      NORP: '#743ccf',
-      PRODUCT: '#d74f57',
-      WORK_OF_ART: '#ffbccc',
-      LANGUAGE: '#d1d3ff',
-    };
-    return colors[group] || '#eee';
-  };
+
 </script>
 
 <div class="sw-entities">
-  {#each entities as group, i}
-    <Collapse expanded={expanded.includes(group[0])}>
-      <button
-        slot="header"
-        on:click={() => {
-          toggle(group[0]);
-        }}
-        class:expanded={expanded.includes(group[0])}
-        class:last={i === entities.length - 1}
+  {#each $entityList as group, i}
+    <Collapse expanded={expanded.includes(group.id)}>
+      <button slot="header"
+              on:click={() => toggle(group.id)}
+              class:expanded={expanded.includes(group.id)}
+              class:last={i === $entityList.length - 1}
       >
-        <div class="color" style:background={getColor(group[0])} />
-        <div class="group-name">{$_('entities.' + group[0].toLowerCase())} ({group[1].length})</div>
+        <div class="color" style:background={group.color}/>
+        <div class="group-name">{$_(group.title)} ({group.entities.length})</div>
+        <div class="icon-container">
+          <Icon name="chevron-left"/>
+        </div>
       </button>
       <ul>
-        {#each group[1] as entity}
-          <li tabIndex="0" role="button" on:click={() => search(entity)} on:keyup={(e) => onKeyUp(e, entity)}>
+        {#each group.entities as entity}
+          <li tabIndex="0" role="button"
+              class:clickable={!$annotationMode}
+              on:click={() => search(entity)}
```

Comment:
A very small issue, click event is still triggered if $`annotationMode` is true

Reply:
good catch, thanks @operramon 
I noticed some bugs on the video widget as well, I fix this in another PR